themes/_core/js/product.js

```diff
@@ -154,6 +154,7 @@ function updateProduct(event, eventType, updateUrl) {
       url: updateUrl + ((updateUrl.indexOf('?') === -1) ? '?' : '&') + formSerialized + preview,
       method: 'POST',
       data: {
+        quickview: parseInt($('.modal.quickview.in').length),
```

Comment:
```suggestion
        quickview: $('.modal.quickview.in').length,
```
parseInt is useless here